workspaces/sonarqube/.changeset/metal-dolphins-serve.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-sonarqube': minor
+---
+
+Added optional `missingAnnotationReadMoreUrl` prop to `SonarQubeContentPageProps` and `SonarQubeCard` to allow configurable links to documentation.
```

Comment:
```suggestion
Added optional `missingAnnotationReadMoreUrl` prop to `SonarQubeContentPage` and `SonarQubeCard` to allow configurable links to documentation.
```

Reply:
@vinzscam fixed in 37145b4cf9af17835dd668962d341881ddb47b5b

Reply:
I mean 8fb5527cc964ba795fe10f7b2f696ed8e36ad0e6 (for the api-report)